src/telemetry-tracing.ts

```diff
@@ -296,81 +310,151 @@ export class PubsubSpans {
   }
 
   static createPublishFlowSpan(message: PubsubMessage): Span | undefined {
-    return PubsubSpans.createChildSpan(message, 'publisher flow control');
+    return PubsubSpans.createChildSpan('publisher flow control', message);
   }
 
-  static createPublishBatchSpan(message: PubsubMessage): Span | undefined {
-    return PubsubSpans.createChildSpan(message, 'publish scheduler');
+  static createPublishSchedulerSpan(message: PubsubMessage): Span | undefined {
+    return PubsubSpans.createChildSpan('publisher scheduler', message);
```

Comment:
```suggestion
    return PubsubSpans.createChildSpan('publish scheduler', message);
```

This is what the C++ impl has (based on the initial design) doc.

I prefer the parallelism of publisher X / publisher Y

@hongalex is there a reason for the publisher scheduler vs. publish scheduler

Reply:
Oh, I suspect that's a typo? I think it should be the same in both cases, too.

Reply:
`publish scheduler` is fine, but will have to update our standard.